frontend/src/components/material/MaterialTable.vue

```diff
@@ -367,10 +376,19 @@ export default {
     isDefaultVariant() {
       return this.clientWidth > 710
     },
+    // Show filter just if period material is in the list
+    periodFilterEnabled() {
+      return this.materialItemCollection.items.some((item) => item.materialNode === null)
+    },
   },
   mounted() {
     this.clientWidth = this.$el.clientWidth
   },
+  updated() {
```

Comment:
You should probably use the `watch` property and not `updated`.
You could watch the `periodFilterEnabled` and set the periodOnly to false if `periodFilterEnabled` is false.

Reply:
Done :)